### ISSUE TITLE
Breaking app

### ISSUE THREAD

**Issue by Shidapu** [Vaz15k/Systemless-AdBlocker#5]
Heya. Please fix human check catchpa for the app https://play.google.com/store/apps/details?id=se.onlinepizza

**Comment by Vaz15k** [comment]
In theory, the captchpa was supposed to go straight to verification...
Can you open the link that is being blocked? The app does not work in my country...

**Comment by Shidapu** [comment]
> In theory, the captchpa was supposed to go straight to verification... Can you open the link that is being blocked? The app does not work in my country...

Yeah true, there is a button saying confirm you are no bot. But there is no link, just text. Maybe a bug in the app? They just recently upgraded the UI.

**Comment by Vaz15k** [comment]
No, you get the link by registering the links that I use, Adaway can do this, you just have to remove the module, use my repository to block ads and use it in VPN mode, then you get the DNS records, activate them and enter the application. (there is an installation method in the readme )
There will be a log on Adaway that will show you the links that were blocked and allowed, just send me the link that refers to the app, I believe that the link will have something related to the name of the app

**Comment by Shidapu** [comment]
> No, you get the link by registering the links that I use, Adaway can do this, you just have to remove the module, use my repository to block ads and use it in VPN mode, then you get the DNS records, activate them and enter the application. (there is an installation method in the readme ) There will be a log on Adaway that will show you the links that were blocked and allowed, just send me the link that refers to the app, I believe that the link will have something related to the name of the app

Alright ill check on it.
